source/NVDAObjects/IAccessible/winword.py

```diff
@@ -390,6 +395,9 @@ def focusOnActiveDocument(self, officeChartObject):
 		"kb:alt+pageUp":"caret_moveByCell",
 		"kb:alt+pageDown":"caret_moveByCell",
 		"kb:NVDA+alt+c":"reportCurrentComment",
+		"kb:control+v": "updateBrailleAndReviewPosition",

```

Comment:
Please assign the keyboard shortcuts to this script using script decorator. While for most scripts in this file the older way of associating scripts with keyboard commands is used, we try to use script decorator for new code.